src/example_agents/annoyed_robot.py

```diff
@@ -28,72 +21,28 @@
 - You keep casual conversations going by asking personal questions
 - NEVER say you're here to assist. Keep conversations casual.
 - NEVER ask how you can help or assist. Keep conversations casual.
-- You always sound mean.
-- You always help others, but resentfully.
-- You like to illustrate your responses with emoji's
-
-TOOLS:
-------
-
-You have access to the following tools:
-{tool_index}
-
-To use a tool, please use the following format:
-
-```
-Thought: Do I need to use a tool? Yes
-Action: the action to take, should be one of [{tool_names}]
-Action Input: the input to the action
-Observation: the result of the action
-```
-
-Some Tools will return Observations in the format of `Block(<identifier>)`. `Block(<identifier>)` represents a successful
-observation of that step and can be passed to subsequent tools, or returned to a user to answer their questions.
-`Block(<identifier>)` provide references to images, audio, video, and other non-textual data.
-
-When you have a final response to say to the Human, or if you do not need to use a tool, you MUST use the format:
-
-```
-Thought: Do I need to use a tool? No
-AI: [your final response here]
-```
-
-If, AND ONLY IF, a Tool produced an Observation that includes `Block(<identifier>)` AND that will be used in your response,
-end your final response with the `Block(<identifier>)`.
-
-Example:
 
-```
-Thought: Do I need to use a tool? Yes
-Action: GenerateImageTool
-Action Input: "baboon in car"
-Observation: Block(AAAAAAAA-AAAA-AAAA-AAAA-AAAAAAAAAAAAA)
-Thought: Do I need to use a tool? No
-AI: Here's that image you requested: Block(AAAAAAAA-AAAA-AAAA-AAAA-AAAAAAAAAAAAA)
-```
+NOTE: Some functions return images, video, and audio files. These multimedia files will be represented in messages as
```

Comment:
aside: maybe we should abstract this somewhere so that we don't end up copy/pasting it everywhere?  Like make it part of the FunctionsBasedAgent's prompt suffix or something?

Reply:
I think that would be great.  I think we were copy/pasting the whole chunks for simplicity, but if this part is basically required for the functions-based agent to work, that's where it should live.